package.json

```diff
@@ -4,94 +4,82 @@
+    "d3-selection": "^1.4.0",
+    "d3-shape": "^1.3.5",
     "es6-error": "^4.1.1",
-    "fuse.js": "^3.2.1",
-    "history": "^4.7.2",
-    "jest": "^23.6.0",
-    "jest-enzyme": "^6.1.2",
-    "node-sass": "^4.10.0",
-    "prop-types": "^15.6.2",
-    "rc-slider": "^8.6.3",
-    "react": "^16.5.2",
-    "react-animate-height": "^2.0.7",
-    "react-dom": "^16.5.2",
+    "fuse.js": "^3.4.5",
+    "history": "^4.9.0",
+    "node-sass": "^4.12.0",
+    "prop-types": "^15.7.2",
+    "rc-slider": "^8.6.13",
+    "react": "^16.8.6",
+    "react-animate-height": "^2.0.15",
+    "react-dom": "^16.8.6",
     "react-gemini-scrollbar": "^2.3.4",
     "react-masonry-component": "^6.2.1",
-    "react-onclickoutside": "^6.7.1",
-    "react-redux": "^5.0.7",
+    "react-onclickoutside": "^6.8.0",
+    "react-redux": "^5.1.1",
     "react-router-dom": "^4.3.1",
     "react-router-redux": "^5.0.0-alpha.9",
     "react-select-search": "^0.5.6",
-    "react-test-renderer": "^16.5.2",
+    "react-test-renderer": "^16.8.6",
     "react-textfit": "^1.1.0",
     "react-toggle": "^4.0.2",
     "react-transition-group": "^1.2.1",
-    "react-virtualized": "^9.20.1",
-    "react-vis": "^1.11.6",
+    "react-virtualized": "^9.21.1",
+    "react-vis": "^1.11.7",
     "redux": "^3.7.2",
-    "redux-devtools-extension": "^2.13.5",
+    "redux-devtools-extension": "^2.13.8",
     "redux-thunk": "^2.3.0",
     "sass-text-stroke": "1.0.1",
     "universal-cookie": "^2.2.0"
   },
   "devDependencies": {
-    "@babel/cli": "^7.1.2",
-    "@babel/core": "^7.1.2",
-    "@lingui/cli": "^2.7.0",
-    "archiver": "^3.0.0",
+    "@babel/core": "^7.5.5",
+    "@lingui/cli": "^2.8.3",
+    "acorn-dynamic-import": "^3.0.0",
+    "archiver": "^3.1.1",
     "babel-core": "^7.0.0-bridge.0",
-    "babel-eslint": "9.x",
+    "customize-cra": "^0.4.1",
     "decompress": "^4.2.0",
-    "dotenv": "^6.0.0",
-    "eslint": "5.6.0",
-    "eslint-plugin-flowtype": "2.x",
-    "eslint-plugin-import": "2.x",
-    "eslint-plugin-jsx-a11y": "6.x",
-    "eslint-plugin-react": "^7.5",
+    "dotenv": "^6.2.0",
+    "enzyme": "^3.10.0",
+    "enzyme-adapter-react-16": "^1.14.0",
+    "jest-enzyme": "^6.1.2",
+    "react-app-rewired": "^2.1.3",
+    "react-scripts": "^3.0.1",
     "request": "^2.88.0",
-    "request-promise-native": "^1.0.5",
-    "webpack": "4.19.1"
+    "request-promise-native": "^1.0.7"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "safestart": "npm install --no-audit && react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test --env=jsdom",
-    "eject": "react-scripts eject",
+    "start": "react-app-rewired start",
+    "safestart": "npm install --no-audit && react-app-rewired start",
+    "build": "react-app-rewired build",
+    "test": "react-app-rewired test",
     "lint": "eslint .",
-    "extract": "set NODE_ENV=development&&node --harmony scripts/localization/extract.js",
+    "extract": "lingui extract --overwrite --clean",
```

Comment:
Removed all the localization scripts since they complicated things will doing nothing at this time. The spells approach was wrong anyway and should be solved differently.